neuralprophet/plot_model_parameters_plotly.py

```diff
@@ -0,0 +1,669 @@
+            name=comp_name,
+            x=range(len(days_i), y=predicted, mode="lines", line=dict(color=color, width=line_width), fill="none"),
+        )
+    )
+
+    xaxis = go.layout.XAxis(title="Day of week")
+    yaxis = go.layout.YAxis(
+        rangemode="normal",
+        title=go.layout.yaxis.Title(text=f"Seasonality: {comp_name}"),
+        zerolinecolor=zeroline_color,
+    )
+
+    if multiplicative:
+        yaxis.update(tickformat="%", hoverformat=".2%")
+
+    return {"traces": traces, "xaxis": xaxis, "yaxis": yaxis}
+
+
+def plot_daily(m, comp_name="daily", quick=True, multiplicative=False):
+    """Plot the daily component of the forecast.
+
+    Args:
+        m (NeuralProphet): fitted model.
+        comp_name (str): Name of seasonality component if previously changed from default 'daily'.
+        weekly_start (int): specifying the start day of the weekly seasonality plot.
+            0 (default) starts the week on Sunday.
+            1 shifts by 1 day to Monday, and so on.
+        quick (bool): use quick low-evel call of model. might break in future.
+        multiplicative (bool): set y axis as percentage
+
+    Returns:
+        A dictionary with Plotly traces, xaxis and yaxis
+    """
+    traces = []
+    color = "#0072B2"
+    line_width = 1
+    zeroline_color = "#AAA"
+
+    # Compute daily seasonality
+    dates = pd.date_range(start="2017-01-01", periods=24 * 12, freq="5min")
+    df = pd.DataFrame({"ds": dates})
+    if quick:
+        predicted = predict_season_from_dates(m, dates=df["ds"], name=comp_name)
+    else:
+        predicted = m.predict_seasonal_components(df)[comp_name]
+
+    traces.append(
+        go.Scatter(
+            name=comp_name,
+            x=range(
+                range(len(dates)), y=predicted, mode="lines", line=dict(color=color, width=line_width), fill="none"
+            ),
+        )
+    )
+
+    xaxis = go.layout.XAxis(title="Hour of day")
+    yaxis = go.layout.YAxis(
+        rangemode="normal",
+        title=go.layout.yaxis.Title(text=f"Seasonality: {comp_name}"),
+        zerolinecolor=zeroline_color,
+    )
+
+    if multiplicative:
+        yaxis.update(tickformat="%", hoverformat=".2%")
+
+    return {"traces": traces, "xaxis": xaxis, "yaxis": yaxis}
+
+
+def plot_custom_season(m, comp_name, multiplicative=False):
+    """Plot any seasonal component of the forecast.
+
+    Args:
+        m (NeuralProphet): fitted model.
+        comp_name (str): Name of seasonality component.
+        multiplicative (bool): set y axis as percentage
+    Returns:
+        A dictionary with Plotly traces, xaxis and yaxis
+    """
+
+    traces = []
+    color = "#0072B2"
+    line_width = 1
+    zeroline_color = "#AAA"
+
+    t_i, predicted = predict_one_season(m, name=comp_name, n_steps=300)
+    traces = []
+
+    traces.append(
+        go.Scatter(
+            name=comp_name,
+            x=range(t_i, y=predicted, mode="lines", line=dict(color=color, width=line_width), fill="none"),
```

Comment:
```suggestion
            x=range(t_i), y=predicted, mode="lines", line=dict(color=color, width=line_width), fill="none",
```
Miss right bracket of range